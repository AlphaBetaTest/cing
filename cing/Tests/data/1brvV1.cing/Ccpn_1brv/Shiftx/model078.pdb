REMARK PDB file of molecule Ccpn_1brv                                 
ATOM      1  N   VAL A 171       3.923   1.979   5.577  1.00  0.00              
ATOM      2  CA  VAL A 171       3.664   2.087   4.121  1.00  0.00              
ATOM      3  HA  VAL A 171       3.281   3.099   3.998  1.00  0.00              
ATOM      4  CB  VAL A 171       5.036   1.998   3.368  1.00  0.00              
ATOM      5  HB  VAL A 171       4.791   2.288   2.343  1.00  0.00              
ATOM      6  CG1 VAL A 171       6.119   3.017   3.793  1.00  0.00              
ATOM      7 HG11 VAL A 171       5.728   4.052   3.782  1.00  0.00              
ATOM      8 HG12 VAL A 171       6.503   2.821   4.812  1.00  0.00              
ATOM      9 HG13 VAL A 171       6.989   2.998   3.112  1.00  0.00              
ATOM     10  CG2 VAL A 171       5.665   0.590   3.291  1.00  0.00              
ATOM     11 HG21 VAL A 171       4.952  -0.150   2.881  1.00  0.00              
ATOM     12 HG22 VAL A 171       6.549   0.573   2.626  1.00  0.00              
ATOM     13 HG23 VAL A 171       5.981   0.222   4.284  1.00  0.00              
ATOM     14  C   VAL A 171       2.530   1.268   3.395  1.00  0.00              
ATOM     15  O   VAL A 171       2.176   1.743   2.315  1.00  0.00              
ATOM     16  N   PRO A 172       1.928   0.124   3.840  1.00  0.00              
ATOM     17  CD  PRO A 172       2.228  -0.464   5.143  1.00  0.00              
ATOM     18  CA  PRO A 172       0.891  -0.632   3.074  1.00  0.00              
ATOM     19  HA  PRO A 172       1.417  -1.175   2.263  1.00  0.00              
ATOM     20  CB  PRO A 172       0.410  -1.659   4.121  1.00  0.00              
ATOM     21  HB2 PRO A 172      -0.453  -1.270   4.697  1.00  0.00              
ATOM     22  HB3 PRO A 172       0.079  -2.608   3.658  1.00  0.00              
ATOM     23  CG  PRO A 172       1.596  -1.852   5.065  1.00  0.00              
ATOM     24  HG2 PRO A 172       1.300  -2.237   6.059  1.00  0.00              
ATOM     25  HG3 PRO A 172       2.319  -2.575   4.636  1.00  0.00              
ATOM     26  HD2 PRO A 172       1.779   0.190   5.920  1.00  0.00              
ATOM     27  HD3 PRO A 172       3.320  -0.508   5.318  1.00  0.00              
ATOM     28  C   PRO A 172      -0.291   0.188   2.467  1.00  0.00              
ATOM     29  O   PRO A 172      -0.891   1.022   3.150  1.00  0.00              
ATOM     30  N   CYS A 173      -0.646  -0.073   1.199  1.00  0.00              
ATOM     31  H   CYS A 173      -0.076  -0.758   0.692  1.00  0.00              
ATOM     32  CA  CYS A 173      -1.680   0.695   0.456  1.00  0.00              
ATOM     33  HA  CYS A 173      -1.372   1.743   0.418  1.00  0.00              
ATOM     34  CB  CYS A 173      -1.579   0.249  -1.004  1.00  0.00              
ATOM     35  HB2 CYS A 173      -1.005  -0.672  -1.148  1.00  0.00              
ATOM     36  HB3 CYS A 173      -2.585   0.105  -1.382  1.00  0.00              
ATOM     37  SG  CYS A 173      -0.675   1.440  -1.999  1.00  0.00              
ATOM     38  C   CYS A 173      -3.112   0.746   1.095  1.00  0.00              
ATOM     39  O   CYS A 173      -3.753   1.801   1.060  1.00  0.00              
ATOM     40  N   SER A 174      -3.577  -0.348   1.728  1.00  0.00              
ATOM     41  H   SER A 174      -2.933  -1.147   1.713  1.00  0.00              
ATOM     42  CA  SER A 174      -4.791  -0.321   2.601  1.00  0.00              
ATOM     43  HA  SER A 174      -5.638  -0.016   1.990  1.00  0.00              
ATOM     44  CB  SER A 174      -5.060  -1.757   3.108  1.00  0.00              
ATOM     45  HB2 SER A 174      -6.031  -1.788   3.640  1.00  0.00              
ATOM     46  HB3 SER A 174      -5.173  -2.451   2.254  1.00  0.00              
ATOM     47  OG  SER A 174      -4.027  -2.234   3.978  1.00  0.00              
ATOM     48  HG  SER A 174      -4.073  -1.711   4.787  1.00  0.00              
ATOM     49  C   SER A 174      -4.789   0.705   3.788  1.00  0.00              
ATOM     50  O   SER A 174      -5.836   1.205   4.197  1.00  0.00              
ATOM     51  N   THR A 175      -3.599   0.995   4.318  1.00  0.00              
ATOM     52  H   THR A 175      -2.811   0.556   3.824  1.00  0.00              
ATOM     53  CA  THR A 175      -3.371   1.977   5.417  1.00  0.00              
ATOM     54  HA  THR A 175      -4.312   2.069   5.994  1.00  0.00              
ATOM     55  CB  THR A 175      -2.317   1.377   6.400  1.00  0.00              
ATOM     56  HB  THR A 175      -2.306   2.014   7.307  1.00  0.00              
ATOM     57  OG1 THR A 175      -0.998   1.379   5.853  1.00  0.00              
ATOM     58  HG1 THR A 175      -1.069   1.271   4.893  1.00  0.00              
ATOM     59  CG2 THR A 175      -2.630  -0.070   6.839  1.00  0.00              
ATOM     60 HG21 THR A 175      -3.678  -0.165   7.184  1.00  0.00              
ATOM     61 HG22 THR A 175      -2.522  -0.776   5.991  1.00  0.00              
ATOM     62 HG23 THR A 175      -1.959  -0.417   7.637  1.00  0.00              
ATOM     63  C   THR A 175      -3.048   3.437   4.929  1.00  0.00              
ATOM     64  O   THR A 175      -3.287   4.392   5.670  1.00  0.00              
ATOM     65  N   CYS A 176      -2.514   3.626   3.700  1.00  0.00              
ATOM     66  H   CYS A 176      -2.556   2.810   3.085  1.00  0.00              
ATOM     67  CA  CYS A 176      -2.354   4.953   3.059  1.00  0.00              
ATOM     68  HA  CYS A 176      -1.856   5.649   3.763  1.00  0.00              
ATOM     69  CB  CYS A 176      -1.404   4.657   1.883  1.00  0.00              
ATOM     70  HB2 CYS A 176      -1.792   3.957   1.150  1.00  0.00              
ATOM     71  HB3 CYS A 176      -1.230   5.534   1.240  1.00  0.00              
ATOM     72  SG  CYS A 176       0.180   4.070   2.513  1.00  0.00              
ATOM     73  C   CYS A 176      -3.722   5.607   2.656  1.00  0.00              
ATOM     74  O   CYS A 176      -3.891   6.818   2.795  1.00  0.00              
ATOM     75  N   GLU A 177      -4.674   4.795   2.148  1.00  0.00              
ATOM     76  H   GLU A 177      -4.310   3.893   1.830  1.00  0.00              
ATOM     77  CA  GLU A 177      -6.098   5.156   1.914  1.00  0.00              
ATOM     78  HA  GLU A 177      -6.437   4.371   1.219  1.00  0.00              
ATOM     79  CB  GLU A 177      -6.893   4.925   3.230  1.00  0.00              
ATOM     80  HB2 GLU A 177      -7.966   4.975   2.970  1.00  0.00              
ATOM     81  HB3 GLU A 177      -6.720   3.884   3.561  1.00  0.00              
ATOM     82  CG  GLU A 177      -6.636   5.880   4.431  1.00  0.00              
ATOM     83  HG2 GLU A 177      -5.583   5.802   4.756  1.00  0.00              
ATOM     84  HG3 GLU A 177      -6.767   6.931   4.115  1.00  0.00              
ATOM     85  CD  GLU A 177      -7.508   5.652   5.664  1.00  0.00              
ATOM     86  OE1 GLU A 177      -8.431   4.847   5.740  1.00  0.00              
ATOM     87  OE2 GLU A 177      -7.145   6.463   6.693  1.00  0.00              
ATOM     88  C   GLU A 177      -6.450   6.471   1.163  1.00  0.00              
ATOM     89  O   GLU A 177      -7.339   7.243   1.535  1.00  0.00              
ATOM     90  N   GLY A 178      -5.764   6.657   0.037  1.00  0.00              
ATOM     91  H   GLY A 178      -4.867   6.164   0.074  1.00  0.00              
ATOM     92  CA  GLY A 178      -5.888   7.889  -0.765  1.00  0.00              
ATOM     93  HA2 GLY A 178      -5.930   7.567  -1.814  1.00  0.00              
ATOM     94  HA3 GLY A 178      -6.845   8.385  -0.570  1.00  0.00              
ATOM     95  C   GLY A 178      -4.758   8.935  -0.637  1.00  0.00              
ATOM     96  O   GLY A 178      -4.796   9.941  -1.347  1.00  0.00              
ATOM     97  N   ASN A 179      -3.744   8.712   0.222  1.00  0.00              
ATOM     98  H   ASN A 179      -3.698   7.742   0.563  1.00  0.00              
ATOM     99  CA  ASN A 179      -2.569   9.598   0.316  1.00  0.00              
ATOM    100  HA  ASN A 179      -2.835  10.631   0.017  1.00  0.00              
ATOM    101  CB  ASN A 179      -1.917   9.483   1.708  1.00  0.00              
ATOM    102  HB2 ASN A 179      -2.655   9.285   2.510  1.00  0.00              
ATOM    103  HB3 ASN A 179      -1.256   8.592   1.643  1.00  0.00              
ATOM    104  CG  ASN A 179      -1.029  10.665   2.061  1.00  0.00              
ATOM    105  OD1 ASN A 179       0.120  10.756   1.648  1.00  0.00              
ATOM    106  ND2 ASN A 179      -1.536  11.613   2.798  1.00  0.00              
ATOM    107 HD21 ASN A 179      -2.494  11.471   3.128  1.00  0.00              
ATOM    108 HD22 ASN A 179      -0.882  12.369   3.013  1.00  0.00              
ATOM    109  C   ASN A 179      -1.576   9.102  -0.722  1.00  0.00              
ATOM    110  O   ASN A 179      -0.906   8.076  -0.560  1.00  0.00              
ATOM    111  N   LEU A 180      -1.463   9.859  -1.790  1.00  0.00              
ATOM    112  H   LEU A 180      -2.112  10.647  -1.857  1.00  0.00              
ATOM    113  CA  LEU A 180      -0.661   9.414  -2.924  1.00  0.00              
ATOM    114  HA  LEU A 180      -1.067   8.399  -3.022  1.00  0.00              
ATOM    115  CB  LEU A 180      -1.079  10.372  -4.051  1.00  0.00              
ATOM    116  HB2 LEU A 180      -1.263  11.389  -3.629  1.00  0.00              
ATOM    117  HB3 LEU A 180      -0.230  10.460  -4.699  1.00  0.00              
ATOM    118  CG  LEU A 180      -2.253   9.941  -4.938  1.00  0.00              
ATOM    119  HG  LEU A 180      -3.044   9.722  -4.224  1.00  0.00              
ATOM    120  CD1 LEU A 180      -2.708  11.088  -5.853  1.00  0.00              
ATOM    121 HD11 LEU A 180      -2.984  11.992  -5.278  1.00  0.00              
ATOM    122 HD12 LEU A 180      -1.920  11.386  -6.571  1.00  0.00              
ATOM    123 HD13 LEU A 180      -3.599  10.809  -6.446  1.00  0.00              
ATOM    124  CD2 LEU A 180      -1.958   8.692  -5.788  1.00  0.00              
ATOM    125 HD21 LEU A 180      -1.682   7.817  -5.172  1.00  0.00              
ATOM    126 HD22 LEU A 180      -2.837   8.388  -6.387  1.00  0.00              
ATOM    127 HD23 LEU A 180      -1.125   8.861  -6.497  1.00  0.00              
ATOM    128  C   LEU A 180       0.875   9.207  -2.736  1.00  0.00              
ATOM    129  O   LEU A 180       1.492   8.402  -3.429  1.00  0.00              
ATOM    130  N   ALA A 181       1.442   9.908  -1.758  1.00  0.00              
ATOM    131  H   ALA A 181       0.706  10.406  -1.259  1.00  0.00              
ATOM    132  CA  ALA A 181       2.808   9.655  -1.255  1.00  0.00              
ATOM    133  HA  ALA A 181       3.478   9.604  -2.125  1.00  0.00              
ATOM    134  CB  ALA A 181       3.202  10.874  -0.415  1.00  0.00              
ATOM    135  HB1 ALA A 181       3.160  11.805  -1.007  1.00  0.00              
ATOM    136  HB2 ALA A 181       2.539  10.993   0.464  1.00  0.00              
ATOM    137  HB3 ALA A 181       4.233  10.761  -0.037  1.00  0.00              
ATOM    138  C   ALA A 181       2.919   8.299  -0.471  1.00  0.00              
ATOM    139  O   ALA A 181       3.757   7.472  -0.819  1.00  0.00              
ATOM    140  N   CYS A 182       2.051   8.047   0.534  1.00  0.00              
ATOM    141  H   CYS A 182       1.227   8.659   0.498  1.00  0.00              
ATOM    142  CA  CYS A 182       1.952   6.733   1.242  1.00  0.00              
ATOM    143  HA  CYS A 182       2.843   6.486   1.781  1.00  0.00              
ATOM    144  CB  CYS A 182       0.942   6.967   2.360  1.00  0.00              
ATOM    145  HB2 CYS A 182       1.112   7.920   2.865  1.00  0.00              
ATOM    146  HB3 CYS A 182      -0.061   7.048   1.945  1.00  0.00              
ATOM    147  SG  CYS A 182       0.930   5.602   3.530  1.00  0.00              
ATOM    148  C   CYS A 182       1.704   5.508   0.311  1.00  0.00              
ATOM    149  O   CYS A 182       2.419   4.504   0.342  1.00  0.00              
ATOM    150  N   LEU A 183       0.699   5.659  -0.544  1.00  0.00              
ATOM    151  H   LEU A 183       0.190   6.548  -0.437  1.00  0.00              
ATOM    152  CA  LEU A 183       0.373   4.739  -1.646  1.00  0.00              
ATOM    153  HA  LEU A 183       0.064   3.768  -1.226  1.00  0.00              
ATOM    154  CB  LEU A 183      -0.825   5.463  -2.295  1.00  0.00              
ATOM    155  HB2 LEU A 183      -0.527   6.491  -2.550  1.00  0.00              
ATOM    156  HB3 LEU A 183      -0.989   5.038  -3.243  1.00  0.00              
ATOM    157  CG  LEU A 183      -2.210   5.440  -1.642  1.00  0.00              
ATOM    158  HG  LEU A 183      -2.056   5.928  -0.663  1.00  0.00              
ATOM    159  CD1 LEU A 183      -3.186   6.234  -2.528  1.00  0.00              
ATOM    160 HD11 LEU A 183      -3.159   5.898  -3.583  1.00  0.00              
ATOM    161 HD12 LEU A 183      -4.230   6.122  -2.200  1.00  0.00              
ATOM    162 HD13 LEU A 183      -2.960   7.314  -2.535  1.00  0.00              
ATOM    163  CD2 LEU A 183      -2.777   4.018  -1.499  1.00  0.00              
ATOM    164 HD21 LEU A 183      -2.114   3.369  -0.910  1.00  0.00              
ATOM    165 HD22 LEU A 183      -3.757   4.007  -0.989  1.00  0.00              
ATOM    166 HD23 LEU A 183      -2.906   3.520  -2.479  1.00  0.00              
ATOM    167  C   LEU A 183       1.517   4.472  -2.693  1.00  0.00              
ATOM    168  O   LEU A 183       1.605   3.372  -3.238  1.00  0.00              
ATOM    169  N   SER A 184       2.355   5.480  -2.989  1.00  0.00              
ATOM    170  H   SER A 184       2.152   6.305  -2.406  1.00  0.00              
ATOM    171  CA  SER A 184       3.583   5.302  -3.829  1.00  0.00              
ATOM    172  HA  SER A 184       3.307   4.719  -4.730  1.00  0.00              
ATOM    173  CB  SER A 184       4.069   6.681  -4.324  1.00  0.00              
ATOM    174  HB2 SER A 184       3.249   7.221  -4.835  1.00  0.00              
ATOM    175  HB3 SER A 184       4.368   7.322  -3.472  1.00  0.00              
ATOM    176  OG  SER A 184       5.163   6.540  -5.236  1.00  0.00              
ATOM    177  HG  SER A 184       5.710   5.811  -4.915  1.00  0.00              
ATOM    178  C   SER A 184       4.742   4.495  -3.135  1.00  0.00              
ATOM    179  O   SER A 184       5.507   3.821  -3.829  1.00  0.00              
ATOM    180  N   LEU A 185       4.910   4.599  -1.805  1.00  0.00              
ATOM    181  H   LEU A 185       4.226   5.222  -1.348  1.00  0.00              
ATOM    182  CA  LEU A 185       5.855   3.731  -1.020  1.00  0.00              
ATOM    183  HA  LEU A 185       6.784   3.565  -1.567  1.00  0.00              
ATOM    184  CB  LEU A 185       6.153   4.395   0.359  1.00  0.00              
ATOM    185  HB2 LEU A 185       5.254   4.270   0.995  1.00  0.00              
ATOM    186  HB3 LEU A 185       6.943   3.810   0.868  1.00  0.00              
ATOM    187  CG  LEU A 185       6.539   5.890   0.401  1.00  0.00              
ATOM    188  HG  LEU A 185       5.697   6.424  -0.080  1.00  0.00              
ATOM    189  CD1 LEU A 185       6.655   6.390   1.848  1.00  0.00              
ATOM    190 HD11 LEU A 185       5.713   6.241   2.407  1.00  0.00              
ATOM    191 HD12 LEU A 185       7.456   5.869   2.407  1.00  0.00              
ATOM    192 HD13 LEU A 185       6.877   7.472   1.888  1.00  0.00              
ATOM    193  CD2 LEU A 185       7.842   6.191  -0.357  1.00  0.00              
ATOM    194 HD21 LEU A 185       7.766   5.906  -1.423  1.00  0.00              
ATOM    195 HD22 LEU A 185       8.087   7.269  -0.338  1.00  0.00              
ATOM    196 HD23 LEU A 185       8.706   5.647   0.067  1.00  0.00              
ATOM    197  C   LEU A 185       5.407   2.259  -0.774  1.00  0.00              
ATOM    198  O   LEU A 185       6.229   1.396  -0.445  1.00  0.00              
ATOM    199  N   CYS A 186       4.109   1.994  -0.909  1.00  0.00              
ATOM    200  H   CYS A 186       3.557   2.837  -0.740  1.00  0.00              
ATOM    201  CA  CYS A 186       3.519   0.723  -0.560  1.00  0.00              
ATOM    202  HA  CYS A 186       3.925   0.565   0.455  1.00  0.00              
ATOM    203  CB  CYS A 186       2.000   1.028  -0.545  1.00  0.00              
ATOM    204  HB2 CYS A 186       1.581   0.319   0.165  1.00  0.00              
ATOM    205  HB3 CYS A 186       1.671   2.002  -0.159  1.00  0.00              
ATOM    206  SG  CYS A 186       1.205   0.785  -2.147  1.00  0.00              
ATOM    207  C   CYS A 186       3.745  -0.569  -1.400  1.00  0.00              
ATOM    208  O   CYS A 186       4.102  -0.603  -2.580  1.00  0.00              
ATOM    209  N   HIS A 187       3.342  -1.643  -0.723  1.00  0.00              
ATOM    210  H   HIS A 187       3.494  -1.510   0.281  1.00  0.00              
ATOM    211  CA  HIS A 187       3.227  -2.996  -1.302  1.00  0.00              
ATOM    212  HA  HIS A 187       3.883  -3.096  -2.185  1.00  0.00              
ATOM    213  CB  HIS A 187       3.676  -4.078  -0.296  1.00  0.00              
ATOM    214  HB2 HIS A 187       2.865  -4.268   0.396  1.00  0.00              
ATOM    215  HB3 HIS A 187       3.774  -5.014  -0.852  1.00  0.00              
ATOM    216  CG  HIS A 187       4.939  -3.835   0.518  1.00  0.00              
ATOM    217  ND1 HIS A 187       4.958  -3.051   1.660  1.00  0.00              
ATOM    218  CD2 HIS A 187       6.211  -3.974  -0.019  1.00  0.00              
ATOM    219  CE1 HIS A 187       6.290  -2.746   1.681  1.00  0.00              
ATOM    220  NE2 HIS A 187       7.135  -3.281   0.744  1.00  0.00              
ATOM    221  HD2 HIS A 187       6.346  -4.378  -1.006  1.00  0.00              
ATOM    222  HE1 HIS A 187       6.608  -1.932   2.312  1.00  0.00              
ATOM    223  C   HIS A 187       1.721  -3.181  -1.696  1.00  0.00              
ATOM    224  O   HIS A 187       0.808  -3.108  -0.865  1.00  0.00              
ATOM    225  N   ILE A 188       1.483  -3.434  -2.980  1.00  0.00              
ATOM    226  H   ILE A 188       2.260  -3.198  -3.589  1.00  0.00              
ATOM    227  CA  ILE A 188       0.119  -3.496  -3.576  1.00  0.00              
ATOM    228  HA  ILE A 188      -0.576  -2.934  -2.913  1.00  0.00              
ATOM    229  CB  ILE A 188       0.246  -2.675  -4.918  1.00  0.00              
ATOM    230  HB  ILE A 188       0.818  -1.746  -4.705  1.00  0.00              
ATOM    231  CG2 ILE A 188       0.990  -3.416  -6.068  1.00  0.00              
ATOM    232 HG21 ILE A 188       1.982  -3.788  -5.752  1.00  0.00              
ATOM    233 HG22 ILE A 188       0.420  -4.289  -6.436  1.00  0.00              
ATOM    234 HG23 ILE A 188       1.168  -2.756  -6.937  1.00  0.00              
ATOM    235  CG1 ILE A 188      -1.142  -2.188  -5.378  1.00  0.00              
ATOM    236 HG12 ILE A 188      -1.733  -3.073  -5.675  1.00  0.00              
ATOM    237 HG13 ILE A 188      -1.650  -1.763  -4.492  1.00  0.00              
ATOM    238  CD1 ILE A 188      -1.161  -1.120  -6.488  1.00  0.00              
ATOM    239 HD11 ILE A 188      -0.573  -0.226  -6.208  1.00  0.00              
ATOM    240 HD12 ILE A 188      -0.750  -1.501  -7.441  1.00  0.00              
ATOM    241 HD13 ILE A 188      -2.192  -0.779  -6.698  1.00  0.00              
ATOM    242  C   ILE A 188      -0.538  -4.921  -3.717  1.00  0.00              
ATOM    243  O   ILE A 188      -1.766  -5.023  -3.705  1.00  0.00              
ATOM    244  N   GLU A 189       0.243  -6.007  -3.858  1.00  0.00              
ATOM    245  H   GLU A 189       1.246  -5.811  -3.881  1.00  0.00              
ATOM    246  CA  GLU A 189      -0.270  -7.385  -4.035  1.00  0.00              
ATOM    247  HA  GLU A 189      -1.068  -7.319  -4.793  1.00  0.00              
ATOM    248  CB  GLU A 189       0.859  -8.252  -4.671  1.00  0.00              
ATOM    249  HB2 GLU A 189       0.381  -9.177  -5.046  1.00  0.00              
ATOM    250  HB3 GLU A 189       1.224  -7.758  -5.593  1.00  0.00              
ATOM    251  CG  GLU A 189       2.084  -8.686  -3.808  1.00  0.00              
ATOM    252  HG2 GLU A 189       1.733  -9.219  -2.905  1.00  0.00              
ATOM    253  HG3 GLU A 189       2.661  -9.443  -4.368  1.00  0.00              
ATOM    254  CD  GLU A 189       3.093  -7.622  -3.363  1.00  0.00              
ATOM    255  OE1 GLU A 189       3.183  -6.489  -3.831  1.00  0.00              
ATOM    256  OE2 GLU A 189       3.913  -8.089  -2.385  1.00  0.00              
ATOM    257  C   GLU A 189      -0.908  -8.013  -2.755  1.00  0.00              
TER     258      GLU A 189 
END   
