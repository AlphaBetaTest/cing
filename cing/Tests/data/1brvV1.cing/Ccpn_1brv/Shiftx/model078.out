# Entries marked with a * may have inaccurate shift predictions.
# Entries marked with a value -666 should be igonored
# Output format adjusted by GV 11 September 2007
  171   V  N      114.0589
  171   V  CA      60.3555
  171   V  CB      34.0917
  171   V  C      175.8018
  171   V  H        8.0549
  171   V  HA       4.0861
  171   V  HB       2.0990
  171   V  HG11     0.9406
  171   V  HG12     0.9132
  171   V  HG13     0.8455
  171   V  HG21     0.9720
  171   V  HG22     0.9505
  171   V  HG23     0.9824
  172   P  CA      62.7672
  172   P  CB      31.7879
  172   P  C      176.2865
  172   P  HA       4.1063
  172   P  HB2      2.2668
  172   P  HB3      2.0311
  172   P  HD2      3.6238
  172   P  HD3      3.8061
  172   P  HG2      2.1399
  172   P  HG3      2.2143
  173   C  N      121.5715
  173   C  CA      60.2015
  173   C  CB      40.7962
  173   C  C      176.2965
  173   C  H        8.3129
  173   C  HA       4.3000
  173   C  HB2      3.1426
  173   C  HB3      3.1406
  174   S  N      114.7714
  174   S  CA      61.4177
  174   S  CB      62.4014
  174   S  C      175.9041
  174   S  H        8.1647
  174   S  HA       3.9954
  174   S  HB2      4.0193
  174   S  HB3      4.0178
  175   T  N      111.7365
  175   T  CA      63.6148
  175   T  CB      69.4938
  175   T  C      176.1375
  175   T  H        7.7543
  175   T  HA       4.2544
  175   T  HB       4.2443
  175   T  HG21     1.2143
  175   T  HG22     1.2170
  175   T  HG23     1.2159
  176   C  N      119.6461
  176   C  CA      57.9366
  176   C  CB      40.7287
  176   C  C      173.5943
  176   C  H        8.2076
  176   C  HA       4.3719
  176   C  HB2      3.1246
  176   C  HB3      3.2166
  177   E  N      116.4763
  177   E  CA      57.4333
  177   E  CB      28.3745
  177   E  C      175.3922
  177   E  H        9.1416
  177   E  HA       3.9680
  177   E  HB2      2.1514
  177   E  HB3      2.0511
  177   E  HG2      2.3613
  177   E  HG3      2.2320
  178   G  N      104.5373
  178   G  CA      46.1766
  178   G  C      173.6397
  178   G  H        8.3918
  178   G  HA    -666.0000
  178   G  HA2      3.9592
  178   G  HA3      3.9588
  179   N  N      118.6581
  179   N  CA      52.4666
  179   N  CB      39.4335
  179   N  C      175.2561
  179   N  H        8.0577
  179   N  HA       4.5912
  179   N  HB2      2.8952
  179   N  HB3      2.7301
  179   N  HD21     7.1313
  179   N  HD22     6.0414
  180   L  N      129.1573
  180   L  CA      57.2946
  180   L  CB      41.6815
  180   L  C      179.4369
  180   L  H        8.4697
  180   L  HA       3.9675
  180   L  HB2      1.7234
  180   L  HB3      1.7012
  180   L  HD11     0.9182
  180   L  HD12     0.9178
  180   L  HD13     0.9181
  180   L  HD21     0.2282
  180   L  HD22     2.0985
  180   L  HD23     0.8667
  180   L  HG       1.7128
  181   A  N      120.7007
  181   A  CA      55.2064
  181   A  CB      18.3389
  181   A  C      179.7114
  181   A  H        7.9038
  181   A  HA       3.9283
  181   A  HB1      1.3867
  181   A  HB2      1.3675
  181   A  HB3      1.3326
  182   C  N      116.0665
  182   C  CA      60.1062
  182   C  CB      43.4917
  182   C  C      175.7971
  182   C  H        7.9453
  182   C  HA       4.1062
  182   C  HB2      3.0295
  182   C  HB3      3.2216
  183   L  N      119.1359
  183   L  CA      57.9786
  183   L  CB      41.2182
  183   L  C      179.2863
  183   L  H        7.9359
  183   L  HA       3.9335
  183   L  HB2      1.8578
  183   L  HB3      1.7205
  183   L  HD11     0.9167
  183   L  HD12     0.9884
  183   L  HD13     0.8999
  183   L  HD21     0.3532
  183   L  HD22     2.1294
  183   L  HD23     0.8664
  183   L  HG       1.7018
  184   S  N      111.9586
  184   S  CA      61.1250
  184   S  CB      62.4087
  184   S  C      175.8243
  184   S  H        7.8235
  184   S  HA       4.2295
  184   S  HB2      4.0080
  184   S  HB3      4.0159
  185   L  N      120.5252
  185   L  CA      55.9880
  185   L  CB      41.5857
  185   L  C      174.9395
  185   L  H        7.6343
  185   L  HA       4.1752
  185   L  HB2      1.7219
  185   L  HB3      1.6925
  185   L  HD11     0.9152
  185   L  HD12     0.9164
  185   L  HD13     0.9160
  185   L  HD21     0.3255
  185   L  HD22     2.8797
  185   L  HD23     0.8624
  185   L  HG       1.7793
  186   C  N      111.9000
  186   C  CA      55.1976
  186   C  CB      40.9534
  186   C  C      172.4526
  186   C  H        7.0781
  186   C  HA       4.4791
  186   C  HB2      3.1415
  186   C  HB3      3.3184
  187   H  N      117.2401
  187   H  CA      54.5601
  187   H  CB      29.2111
  187   H  C      175.9810
  187   H  H        8.3186
  187   H  HA       4.7147
  187   H  HB2      3.2744
  187   H  HB3      3.2833
  187   H  HD2      5.7975
  187   H  HE1      6.6833
  188   I  N      126.9424
  188   I  CA      62.8202
  188   I  CB      38.8716
  188   I  C      175.7111
  188   I  H        8.4119
  188   I  HA       4.0505
  188   I  HB       1.8640
  188   I  HD11     0.8549
  188   I  HD12     0.8579
  188   I  HD13     0.8582
  188   I  HG12     1.2747
  188   I  HG13     0.8006
  188   I  HG21     0.9129
  188   I  HG22     0.9134
  188   I  HG23     0.9086
 *189   E  N      119.9175
 *189   E  CA      55.9946
 *189   E  CB      30.2534
 *189   E  C      179.1042
 *189   E  H        8.0703
 *189   E  HA       4.1706
 *189   E  HB2      1.7813
 *189   E  HB3      1.7575
 *189   E  HG2      2.1386
 *189   E  HG3      2.3839
